app/ViewLocations.tsx

```diff
@@ -0,0 +1,44 @@
+import { ReactElement } from "react";
+import { Wrapper } from "./ui/Wrapper";
+import { styled } from "styled-components";
+
+const locations = [
+  {
+    displayName: "TANG",
+    fullName: "TANG - The Asian Food Emporium",
+    suburb: "Melbourne",
+  },
+  {
+    displayName: "Minh Phat",
+    fullName: "Minh Phat Asian Grocer",
+    suburb: "Richmond",
+  },
+  {
+    displayName: "Aldi",
+    fullName: "ALDI",
+    suburb: "Richmond",
+  },
+];
+
+export function ViewLocations(): ReactElement {
+  const rowSeed = locations.map((location) => {
```

Comment:
i know you copied this from the other project but it's a weird name. You can also do this inline below so you don't need a name at all, like

```
<Wrapper>
{
locations.map((location) => {
    return (
      <div style={{ display: "flex" }}>
        <Cell style={{ color: "black" }}>{location.displayName}</Cell>
        <Cell style={{ color: "#535353" }}>{location.suburb}</Cell>
      </div>
    );
  });
}
</Wrapper>
```

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/12/commits/706b68b72dcfc87e2f51c92ad51d17589a6f5460